### PR TITLE
MWAA Edits

### PR DESCRIPTION
**Pull request by lzdanski**
* Editorial review of MWAA migration doc and partials
* Copy editing + Style guide changes

### CODE REVIEW

File: astro/migration-partials/workspaces.mdx [hunk 1]
```diff
@@ -1,8 +1,8 @@
-Within your Astro Organization you can create _Workspaces_, which are collections of Airflow environments that are typically owned by a single team. 
+In your Astro Organization, you can create _Workspaces_, which are collections of Airflow environments. Workspaces are typically owned by a single team. 
 
-Follow the steps in [Manage Workspaces](manage-workspaces.md) to create a Workspace in the Astro UI for your migrated Airflow environments. Astronomer recommends naming your first Workspace after your data team or initial business use case with Airflow. You can update these names in the Cloud UI after you finish the migration.
```
Comment (lzdanski) [inline astro/migration-partials/workspaces.mdx:3]: Not sure if these are unrelated actions or a sequence of actions.

File: astro/migration-partials/starship-usage.mdx [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 - Environment variables
 - A secrets backend.
 
-If you set your Airflow variables and connections in the Airflow UI, you can migrate those to Astro with Starship. You can check which resources will be migrated by going to **Admin** > **Variables** and **Admin** > **Connections** in the Airflow UI for your source Airflow environment.
+If you defined your Airflow variables and connections in the Airflow UI, you can migrate those to Astro with Starship. You can check which resources will be migrated by going to **Admin** > **Variables** and **Admin** > **Connections** in the Airflow UI to find your source Airflow environment information.
 
 :::caution
 
-Some environment variables or Airflow Settings can't be migrated to Astro. See [Global environment variables](https://docs.astronomer.io/astro/platform-variables) for a list of variables that you can't override on Astro.
+Some environment variables or Airflow Settings can't be migrated to Astro. See [Global environment variables](https://docs.astronomer.io/astro/platform-variables) for a list of variables that you can't migrate to Astro.
```
Comment (lzdanski) [inline astro/migration-partials/starship-usage.mdx:14]: This sentence didn't make sense to me. Why are the variables that you can't migrate also ones that can't be overridden on astro.

File: astro/migration-partials/starship-usage.mdx [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
-You might have set Airflow connections and variables in the following places on your source Airflow environment:
+You might have defined Airflow connections and variables in the following places on your source Airflow environment:
 
 - The Airflow UI (stored in the Airflow metadata database).
 - Environment variables
 - A secrets backend.
 
-If you set your Airflow variables and connections in the Airflow UI, you can migrate those to Astro with Starship. You can check which resources will be migrated by going to **Admin** > **Variables** and **Admin** > **Connections** in the Airflow UI for your source Airflow environment.
+If you defined your Airflow variables and connections in the Airflow UI, you can migrate those to Astro with Starship. You can check which resources will be migrated by going to **Admin** > **Variables** and **Admin** > **Connections** in the Airflow UI to find your source Airflow environment information.
```
Comment (lzdanski) [inline astro/migration-partials/starship-usage.mdx:10]: `set` was kind of ambiguous to my first read through, defined? configured?

File: astro/migration-partials/prereqs.mdx [hunk 4]
```diff
@@ -1,8 +1,8 @@
-The migration process should be completed with assistance from your Astronomer Data Engineer. Before you meet with your Astronomer Data Engineer, ensure that the following steps are completed:
+The migration process should be completed with assistance from your Astronomer Data Engineer. Before you meet with your Astronomer Data Engineer, complete the following steps:
 
-- Astronomer has [activated your installation](https://docs.astronomer.io/astro/category/install-astro).
```
Comment (lzdanski) [inline astro/migration-partials/prereqs.mdx:3]: Changed a lot of past perfect sentence constructions to simple past - easier to read and understand for non English as first language folks.

File: astro/migrate-mwaa.mdx [hunk 5]
```diff
@@ -154,17 +152,21 @@ If you used CI/CD to deploy code to your source Airflow environment, read the fo
...
 
-Depending on how thoroughly you want to test your Airflow environment, you have a few options for testing your project locally before deploying to Astro.
+Depending on how thoroughly you want to test your Airflow environment, you can test your project locally before deploying to Astro.
 
 - In your Astro project directory, run `astro dev parse` to check for any parsing errors in your DAGs.
 - Run `astro run <dag-id>` to test a specific DAG. This command compiles your DAG and runs it in a single Airflow worker container based on your Astro project configurations.
 - Run `astro dev start` to start a complete Airflow environment on your local machine. After your project starts up, you can access the Airflow UI at `localhost:8080`. See [Test and troubleshoot locally](test-and-troubleshoot-locally.md).
 
-Note that your migrated Airflow variables and connections are not available locally. You must deploy your project to Astro to test these resources. 
+:::tip
```
Comment (jwitz) [inline astro/migrate-mwaa.mdx:165]:
```suggestion
:::info
```

**Review by jwitz — APPROVED** [review]
LGTM! Just a few small notes

File: astro/migration-partials/deployments.mdx [hunk 6]
```diff
@@ -49,7 +49,7 @@ Before you create your Deployment, copy the following information from your sour
     | Medium (Up to ~250 DAGs) | 10  | 1vCPU, 3.75GiB*           |
     | Large (Up to ~1000 DAGs) | 15  | 1.5vCPU, 5.64GiB*         |
 
-    > *Some of the following recommendations for CPU and memory are smaller than what you would have in the equivalent source environment. Although you may not need more CPU or memory than what's recommended here, some environments might parse DAGs slower than in your source Airflow environment to start. Use these recommendations as a starting point, then adjust your resource usage after tracking your performance on Astro.
+    > *Some of the following recommendations for CPU and memory are smaller than what you have in the equivalent source environment. Although you might not need more CPU or memory than what's recommended. Some environments might parse DAGs slower than your source Airflow environment to start. Use these recommendations as a starting point, then adjust your resource usage after tracking your performance on Astro.
```
Comment (lzdanski) [inline astro/migration-partials/deployments.mdx:52]:
```suggestion
    > *Some of the following recommendations for CPU and memory are smaller than what you have in the equivalent source environment. Although you might not need more CPU or memory than what's recommended, some environments might parse DAGs slower than your source Airflow environment to start. Use these recommendations as a starting point, then adjust your resource usage after tracking your performance on Astro.
```

File: astro/migration-partials/workspaces.mdx [hunk 7]
```diff
@@ -1,8 +1,8 @@
-Within your Astro Organization you can create _Workspaces_, which are collections of Airflow environments that are typically owned by a single team. 
+In your Astro Organization, you can create _Workspaces_, which are collections of Airflow environments. Workspaces are typically owned by a single team.
```
Comment (paolaperaza) [inline astro/migration-partials/workspaces.mdx:1]: Correct def of workspace to match QuickStart

File: astro/migration-partials/workspaces.mdx [hunk 8]
```diff
@@ -1,8 +1,8 @@
-Within your Astro Organization you can create _Workspaces_, which are collections of Airflow environments that are typically owned by a single team. 
+In your Astro Organization, you can create _Workspaces_, which are collections of Airflow environments. Workspaces are typically owned by a single team. 
 
-Follow the steps in [Manage Workspaces](manage-workspaces.md) to create a Workspace in the Astro UI for your migrated Airflow environments. Astronomer recommends naming your first Workspace after your data team or initial business use case with Airflow. You can update these names in the Cloud UI after you finish the migration.
+1. Follow the steps in [Manage Workspaces](manage-workspaces.md) to create a Workspace in the Astro UI for your migrated Airflow environments. Astronomer recommends naming your first Workspace after your data team or initial business use case with Airflow. You can update these names in the Cloud UI after you finish the migration.
```
Comment (paolaperaza) [inline astro/migration-partials/workspaces.mdx:3]: Cloud UI not Astro

File: astro/migration-partials/workspaces.mdx [hunk 9]
```diff
@@ -11,6 +11,6 @@ You can add users to a Workspace an Organization using the Astro CLI. See:
 - [`astro workspace user add`](cli/astro-workspace-user-add.md) 
 - [`astro organization user invite`](cli/astro-organization-user-invite.md)
 
-Using shell scripting, you can automate adding batches of users to Astro. See [Add a group of users to Astro using the Astro CLI](add-user.md#add-a-group-of-users-to-astro-using-the-astro-cli).
+:::
 
-:::
\ No newline at end of file
+Using shell scripting, you can automate adding batches of users to Astro. See [Add a group of users to Astro using the Astro CLI](add-user.md#add-a-group-of-users-to-astro-using-the-astro-cli).
```
Comment (paolaperaza) [inline astro/migration-partials/workspaces.mdx:16]: Shell script as noun not verb?

**Review by paolaperaza — APPROVED** [review]
Couple comments but SG @lzdanski !

File: astro/migration-partials/deploy.mdx [hunk 10]
```diff
@@ -12,7 +12,7 @@
 
     This command returns a list of Deployments available in your Workspace and prompts you to pick one.
 
-    After you select a Deployment, the CLI parses your DAGs to ensure that they don't contain basic syntax and import errors. This test is equivalent to the one that runs during `astro dev parse` in a local Airflow environment. If any of your DAGs fail this parse, the deploy to Astro also fails.
+    After you select a Deployment, the CLI parses your DAGs to check for basic syntax and import errors. This test is equivalent to the one that runs during `astro dev parse` in a local Airflow environment. If any of your DAGs fail this parse, the deploy to Astro also fails.
```
[the remark this answers is not in the document]
Comment (lzdanski) [reply astro/migration-partials/deploy.mdx:15]: We removed this from the quickstart too, so I'll take it out
